install_config/aggregate_logging.adoc

```diff
@@ -66,16 +66,15 @@ xref:../install_config/persistent_storage/index.adoc#install-config-persistent-s
 necessary storage] for Elasticsearch. Note that each Elasticsearch replica
 requires its own storage volume. See
 xref:aggregated-elasticsearch[Elasticsearch] for more information.
-. Choose a project. Once deployed, the EFK stack collects logs for every
+. Once deployed, the EFK stack collects logs for every
```

Comment:
I'd s/Once deployed/After it deploys

Will you consider changing this list to an unordered list instead of an ordered list or making all the steps explicit?